staging/src/k8s.io/apiserver/pkg/server/options/feature.go

```diff
@@ -43,9 +45,12 @@ func (o *FeatureOptions) AddFlags(fs *pflag.FlagSet) {
 	}
 
 	fs.BoolVar(&o.EnableProfiling, "profiling", o.EnableProfiling,
-		"Enable profiling via web interface host:port/debug/pprof/")
+		"Enable profiling via web interface host:port/debug/pprof/, or unix-domain socket if --profiling-path is specified.")
```

Comment:
I'd keep enabling it for real vs on the uds socket totally separate flags.

Reply:
+1, or a toggle option between off, http, uds but for backwards compatibility a separate flag is probably better.